instance/tests/models/test_openedx_appserver.py

```diff
@@ -544,6 +546,32 @@ def test_make_active(self, mocks):
         self.assertFalse(instance.get_active_appservers().exists())
         self.assertEqual(mocks.mock_load_balancer_run_playbook.call_count, 3)
         self.assertEqual(mocks.mock_disable_monitoring.call_count, 0)
+        self.assertEqual(mocks.mock_run_appserver_playbooks.call_count, 2)
+
+    @patch_services
+    def test_make_active_fails_to_start_services(self, mocks):
+        """
+        Test make_active() and check if its behaving correctly when the
+        playbooks to start services fail
+        """
+        mocks.mock_run_appserver_playbooks.return_value = ('', 2)
+        instance = OpenEdXInstanceFactory(internal_lms_domain='test.activate.opencraft.co.uk')
+        appserver_id = instance.spawn_appserver()
+        self.assertEqual(mocks.mock_load_balancer_run_playbook.call_count, 1)
+        appserver = instance.appserver_set.get(pk=appserver_id)
+
+        self.assertEqual(instance.appserver_set.get().last_activated, None)
+
+        appserver.make_active()
```

Comment:
Is this supposed to fail?

Reply:
No, it should simply return without modifying the appserver.
Do you think its better if we raise an exception?

Reply:
It would be good to have some indication if it failed. Maybe return True/False?

Reply:
The other methods below don't return anything, so the behavior won't be consistent if I return `true/false`.